db/migrate/20170207192912_encrypt_otp_secret.rb

```diff
@@ -0,0 +1,22 @@
+class EncryptOtpSecret < ActiveRecord::Migration
+  class User < ActiveRecord::Base
+  end
+
+  def up
+    encrypt_otp_secret_key
+  end
+
+  def down
+  end
+
+  def encrypt_otp_secret_key
+    User.where.not(otp_secret_key: nil).find_in_batches.with_index do |users, batch|
+      users.each do |user|
+        encrypted_attribute = EncryptedAttribute.new_from_decrypted(user.otp_secret_key)
```

Comment:
Good point, but replacing a migration with a script does not solve that problem either, right? The script, written today, will still need to reference `EncryptedAttribute` as implemented today. Or is there a way around that?

I think the solution to worked towards is being able to deploy often to our servers. 

If there comes a time when `EncryptedAttribute` needs to be renamed or changed, we'll need to make sure to either keep the old version around for those servers that haven't yet run the migration, or update the migration.